pairing/bls12381/kilic/g1.go

```diff
@@ -0,0 +1,172 @@
+package kilic
+
+import (
+	"bytes"
+	"crypto/cipher"
+	"encoding/hex"
+	"io"
+
+	bls12381 "github.com/kilic/bls12-381"
```

Comment:
Given the fact that https://github.com/kilic/bls12-381 is now archived, I'm starting to have second thoughts about including an adapter for it in Kyber.
WDYT @pierluca @ineiti ?

Reply:
You could:
- fork it into dedis/bls12-381 to avoid it missing if the author removes it
- effectively drop support for it

I have to admit I'm not sure if somebody is using it. But if it's not too big a hassle, I think the goal of kyber should be to have as many algorithms as possible.

So my proposition is to do nothing here - or perhaps add a comment somewhere and writing what you decided and how.